tests/lib/assertions.py

```diff
@@ -159,19 +159,34 @@ def assert_200_valid_message_id_response_body(resp, message_id, url):
         assert actual == expected
 
     @staticmethod
-    def assert_message_delivered_gov_uk(response, message_id, type):
+    def assert_email_gov_uk(response, message_id):
         for i in range(len(response)):
             if message_id in response[i].get("reference"):
                 assert response[i].get("status") == "delivered"
-                assert response[i].get("type") == type
+                assert response[i].get("type") == "email"
                 assert response[i].get("body") is not None
-                if type == "email":
-                    assert response[i].get("subject") is not None
-                    assert response[i].get("email_address") is not None
-                if type == "sms":
-                    assert response[i].get("phone_number") is not None
+                assert response[i].get("subject") is not None
+                assert response[i].get("email_address") is not None
+
+    @staticmethod
+    def assert_sms_gov_uk(response, message_id):
+        for i in range(len(response)):
+            if message_id in response[i].get("reference"):
+                assert response[i].get("status") == "delivered"
+                assert response[i].get("type") == "sms"
+                assert response[i].get("body") is not None
+                assert response[i].get("phone_number") is not None
                 break
 
+    @staticmethod
+    def assert_letter_gov_uk(response, message_id):
+        for i in range(len(response)):
```

Comment:
I appreciate you're just copying a pattern and if you're content with copying this pattern no problemo, but just checking you are aware that indexable objects in python are also often iterable, so you can just do `for message in response:` and then `assert message.get("status") == "received"` and so on and so forth. Might look a bit nicer?

Reply:
Looks a load better, cheers for the suggestion!